### ISSUE TITLE
Reproducibility test for model creation fails

### ISSUE THREAD

**Issue by Justin-Sonneck** [bioimage-io/core-bioimage-io-python#345]
I would like to create a BioImage.IO model and follow the provided notebook to do so, but as soon as I use a model other than the provided few convolutions, the reproducibility test within the notebook fails, although the code runs successfully and I do not make any changes to the notebook except for the model.

In my case I replace the convolutions with MONAI's UNet, here's the changed code: 
```
# a very simple pytorch model: just a few convolutions
# model = nn.Sequential(
# nn.Conv2d(1, 16, 3),
# nn.Conv2d(16, 32, 3),
# nn.Conv2d(32, 16, 3),
# nn.Conv2d(16, 1, 1)
# )

model = monai.networks.nets.UNet(2,1,1,[32,64,128,256], [2,2,2])
model = torch.jit.script(model)

# save the model weights
model.save("my-model/weights.pt")
```

I made sure the stored output is the prediction of the UNet model and not that of the provided model, but the reproducibility test shows:
```
{'name': 'reproduce test outputs from test inputs (bioimageio.core 0.5.9)',
  'status': 'failed',
  'error': 'Output and expected output disagree:\n \nArrays are not almost equal to 4 decimals\n\nMismatched elements: 14125 / 16384 (86.2%)\nMax absolute difference: 0.00538778\nMax relative difference: 4.027447\n x: array([[[[-0.4896, -0.2421,  0.0273, ..., -0.2012, -0.5665,  1.0788],\n         [ 0.4324, -1.7349,  0.611 , ..., -1.3215, -1.0914, -1.7692],\n         [ 0.2386,  2.2838, -0.3978, ..., -0.4865, -2.1575, -0.4432],...\n y: array([[[[-0.4889, -0.2425,  0.0275, ..., -0.2015, -0.5664,  1.0788],\n         [ 0.4327, -1.7323,  0.6103, ..., -1.322 , -1.0921, -1.7693],\n         [ 0.2392,  2.2836, -0.3976, ..., -0.4864, -2.1575, -0.4429],...',
  'traceback': None,
  'bioimageio_spec_version': '0.4.9',
  'bioimageio_core_version': '0.5.9',
  'warnings': {},
  'source_name': 'MyFirstModel'}]
  ```
  

Does anyone know how I can solve this? Thanks in advance!

**Comment by FynnBe** [comment]
> Max absolute difference: 0.00538778\nMax relative difference: 4.027447\

Our default test comparison accuracy is 4 decimals, to pass the test for your scripted model you'd have to set it to 2 decimals. 
Alternatively you could of course try to improve the numerical 'similarity'. See https://pytorch.org/docs/stable/notes/numerical_accuracy.html# for some general notes, but that would probably be very tedious.

We have not yet documented an official way of altering the test accuracy, but it is available as an 'experimental' feature:
Add this to the rdf.yaml's `config` field:
```yaml
config:
  bioimageio:
    test_kwargs:
      torchscript:
        decimal: 2
```

**Comment by FynnBe** [comment]
FYI: The comparison is done with https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_almost_equal.html

> abs(desired-actual) < 1.5 * 10**(-decimal)

**Comment by Justin-Sonneck** [comment]
Ahh, I see. Thanks for the quick support!
